packages/presentation/lib/config/theme/app_colors.dart

```diff
@@ -1,10 +1,12 @@
 import 'package:flutter/material.dart';
 
-class AppColors {
+class AppColorsDark {
   static const primaryColorDark = Color.fromRGBO(15, 27, 43, 1);
   static const canvasColor = Color.fromRGBO(15, 27, 43, 1);
   static const redTabBar = Color.fromRGBO(217, 37, 29, 1);
```

Comment:
in case when colors connected to theme u should avoid specify color name (like REDtabBar). it should be tabBarColor or smth like this

Reply:
edited